Tools/Drone/etc/hosts

```diff
@@ -0,0 +1,9 @@
+127.0.0.1       localhost
+127.0.1.1       drone4 drone4.local
```

Comment:
does $(hostname) not work here?

Reply:
I sadly could not get something like this to work. Let me see if I can figure out a way, as this is definitely the better way to handle the file, if it is possible.